README.md

```diff
@@ -81,6 +81,29 @@ Currently the component that takes the longest time to compile is
 `ihaskell-widgets`, so if you're in a hurry you may want to comment that out in
 `stack.yaml`.
 
+## Stack development with Docker
+This is an alternative way to use Docker than above, taking advantage of stack's Docker support.
+Makes it easy to install new packages with `stack --docker install <pkg>` without having to rebuild a docker image.
+Like the other Docker workflow, this doesn't require any host dependecies to be installed.
+
+```
+docker build -t ihaskell-dev docker
+stack --docker setup
+stacl --docker install
+stack --docker exec ihaskell -- install --stack
+stack --docker exec jupyter -- notebook --ip=0.0.0.0 notebooks
+```
+
+Everything in the LTS should be available!
```

Comment:
I don't see how this is true. Does the `Dockerfile` somehow download and install all packages in the LTS? If not, please reword this to say something like "Everything in the LTS can be made available!"

Reply:
Agree, "can be made available" is more accurate